Intersections_3/include/CGAL/internal/Intersections_3/Bbox_3_Triangle_3_do_intersect.h

```diff
@@ -54,7 +54,8 @@ namespace internal {
         }
         else {
           if(p[i] <= r[i]) { // prq
-            if((bbox.max)(i) < p[i] || (bbox.min)(i) > q[i])
+            if(
+               (bbox.max)(i) < p[i] || (bbox.min)(i) > q[i])
```

Comment:
?

Reply:
?????

Reply:
? == Why a newline

Reply:
????? == What the hell is this change doing here ?